pypopquiz/io.py

```diff
@@ -90,6 +90,14 @@ def read_input(file_name: Path) -> Dict:
         return input_data
 
 
+def write_lines(text: Iterable[str], file_name: Path) -> None:
+    """Writes a list of items to file"""
+    with file_name.open("w") as file:
```

Comment:
A shorter alternative would be:
```python
file_name.write_text('\n'.join(text))
```
It doesn't have the final \n without additional ugliness, but it is quite compact. No dealbreaker, choose what you like best.

Reply:
Done!